app/views/insured/consumer_roles/_no_match.html.haml

```diff
@@ -1,3 +1,3 @@
 .col-md-12.col-sm-12.col-xs-12.no-pd
   .alert.alert-success.alert-dismissible
-    = "Thank you. Next, we need to verify if you or you and your family are eligible to enroll in coverage through #{site_short_name}. Please select CONTINUE."
+    = l10n("insured.consumer_roles.no_match_info", site_short_name: site_short_name)
```

Comment:
@cdeyoung can you confirm - this hierarchy is not needed ?
```
l10n('.no_match_info')
```
would find the equivalent value right?